src/middlewares/withAuth.ts

```diff
@@ -14,48 +16,54 @@ export function withAuth(handler: Handler): Handler {
     request: ExtendedRequest,
     response: ServerResponse,
   ): Promise<unknown> => {
-    const userSession = request.session.get<UserCookie | undefined>(
+    const userCookie = request.session.get<UserCookie | undefined>(
       "user-session",
     );
```

Comment:
We discussed the naming with Tom and Vincent, because it was previously named `userSession`, which bothered Tom.

`userCookie` is the decrypted version of `user-session`, without the "session" prefix, so it's not the same.

Reply:
Ok!
But what about the name of the cookie itself, still `"user-session"`?

Reply:
You are right, I'll change this too.

Reply:
Actually, I'll do this in an other PR, this will change a lot of stuff.

Reply:
👍